react/Viewer/index.jsx

```diff
@@ -149,6 +151,19 @@ const ViewerInformationsWrapper = ({
   validForPanel,
   toolbarRef
 }) => {
+  const theme = useTheme()
+  const sidebar = document.querySelector('[class*="sidebar"]')
+
+  useSetFlagshipUI(
```

Comment:
nit: add a test checking useSetFlagshipUI is called with correct arguments, when this component is rendered

Reply:
well useSetFlagshipUI is trying to be as declarative as CSS. I suggest an alternative, adding static tests to useSetFlagshipUI and verify that incomplete arguments still work.

I don't think we should test that component XYZ is using values red, blue and green for instance

Reply:
https://github.com/cozy/cozy-ui/pull/2072/files#diff-4cbb730f55f4a37e3fa25231e3f79b6f1d5901fa59e89ec6455568fc400be2fdR1